css/store.css

```diff
@@ -0,0 +1,238 @@
+/* store page */
+
+/* main */
+.storemain {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+/* section one */
+.formsection {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 30px;
+}
+/* form */
+form {
```

Comment:
Better to style classes rather than elements, so that you don't get unwanted styles applied to all of the elements